config/config.exs

```diff
@@ -1,6 +1,6 @@
 # This file is responsible for configuring your application
 # and its dependencies with the aid of the Mix.Config module.
-use Mix.Config
+import Config
```

Comment:
The problem seems to be that this fails the build on Elixir versions [1.8 and below](https://github.com/jonasschmidt/ex_json_schema/pull/78/checks?check_run_id=7934395921). Is that change necessary?

Reply:
I have reverted it. I was trying to get to a clean build. 

Mix.Config was deprecated in Elixir 1.9